cmd/konvoy-image-wrapper/cmd/wrapper.go

```diff
@@ -186,6 +189,36 @@ func (r *Runner) setAzureEnv() {
 	}
 }
 
+func (r *Runner) setGCPEnv() error {
+	for _, env := range []string{
+		envGCPApplicationCredentials,
+	} {
+		value, found := os.LookupEnv(env)
+		if found {
+			r.env[env] = value
+		}
+	}
+
+	for _, env := range []string{
+		envGCPApplicationCredentials,
```

Comment:
not sure why looping through list of one member. are we anticipating more such variables here?
Also we can combine logic here:
```go
path, found := os.LookupEnv(envGCPApplicationCredentials)
if found {
			if fi, err := os.Stat(path); err == nil {
				if fi.IsDir() {
					return fmt.Errorf("env %s must be set to a file", env)
				}
                                 r.env[env] = value
				// bind to exact same path
				r.addBindVolume(path, path)
                                 return nil
			} 
                         // something like below. please rephrase
			return fmt.Errorf("error accessing %q: %v", path, err)
}
return fmt.Errorf("required env variable %s not set", envGCPApplicationCredentials)

```

Reply:
Thanks for pointing. It makes sense.